### PR TITLE
Cherry pick into 2.5.1

### PR DESCRIPTION
**Pull request by javierguerragiraldez**

### CODE REVIEW

File: kong/plugins/datadog/handler.lua [hunk 1]
```diff
@@ -89,7 +89,7 @@ end
 
 local DatadogHandler = {
   PRIORITY = 10,
-  VERSION = "3.0.1",
+  VERSION = "3.1.0",
```
Comment (mikefero) [inline kong/plugins/datadog/handler.lua:92]: What is changing besides the version; I don't see any underlying plugin changes here or in the `release/2.5.x` branch from the last `2.5.0` release.

File: kong/plugins/request-termination/handler.lua [hunk 2]
```diff
@@ -15,7 +15,7 @@ local RequestTerminationHandler = {}
 
 
 RequestTerminationHandler.PRIORITY = 2
-RequestTerminationHandler.VERSION = "2.0.1"
+RequestTerminationHandler.VERSION = "2.1.0"
```
Comment (mikefero) [inline kong/plugins/request-termination/handler.lua:18]: Same here for the request termination plugin.

Comment (javierguerragiraldez) [reply kong/plugins/request-termination/handler.lua]: (commit removed)

File: kong/plugins/datadog/handler.lua (hunk 1 above)
Comment (javierguerragiraldez) [reply kong/plugins/datadog/handler.lua]: (commit removed)

**Comment by akira28** [comment]
Sorry for the "rude" question, do you have any timeline for release 2.5.1? At the moment we are using an ugly workaround for the import/export bug, and we would like to avoid it if possible.
Thanks for the great work!

**Comment by javierguerragiraldez** [comment]
> do you have any timeline for the release 2.5.1?

it should go out tomorrow Sept 10

**Pull request merged**